configs/98netflix

```diff
@@ -0,0 +1,6 @@
+#!/system/bin/sh
```

Comment:
no, it will not, ishant has refused to maintain it as a official device, also we still don't know who made request for it to be a official device, no official device use patches and custom webview, think about that